pms-migration-utility-default.properties

```diff
@@ -64,7 +68,8 @@ hibernate.generate_statistics=false
 spring.datasource.initialization-mode=never
 
 # websub related properties
-websub.publish.url=mosip.websub.url
+mosip.websub.url=http://websub.websub
```

Comment:
I think this property is already there in application-default.properties, why do we need this again here ?